repos/spack_repo/builtin/packages/libmetatomic_torch/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+
+from spack.package import *
+
+
+class LibmetatomicTorch(CMakePackage):
+    """TorchScript/C++ bindings to metatomic"""
+
+    homepage = "https://docs.metatensor.org/metatomic"
+    url = "https://github.com/metatensor/metatomic/releases/download/metatomic-torch-v0.1.3/metatomic_torch-0.1.3.tar.gz"
```

Comment:
```suggestion
    url = "https://github.com/metatensor/metatomic/releases/download/metatomic-torch-v0.1.4/metatomic-torch-cxx-0.1.4.tar.gz"
```
Update to `0.1.4`? This should also directly tap into the new CXX artifact.

Reply:
This also requires updating `libmetatensor-torch` to `0.8.0`.